runtime/doc/starting.txt

```diff
@@ -90,10 +90,13 @@ argument.
 		See |info-message| about capturing the text.
 
 							*--clean*
---clean		Equivalent to "-u NONE -i NONE":
+--clean		Similar to "-u NONE -i NONE":
 		- Skips initializations from files and environment variables.
 		- No 'shada' file is read or written.
 		- Excludes user directories from 'runtimepath'
+		- Unlike "-u NONE -i NONE" plugins are still loaded. This
+		  means "--clean" effectively mimics a fresh install of Nvim
+		  with no user configuration.
```

Comment:
cc @justinmk 

Happy to iterate on the wording.

Reply:
"**built-in** plugins are still loaded" or "**default** plugins" (still not ideal, though)